cpp/src/arrow/compute/kernels/aggregate_test.cc

```diff
@@ -744,20 +771,10 @@ TYPED_TEST(TestMeanKernelNumeric, ScalarAggregateOptions) {
 
   ValidateMean<TypeParam>("[]", nan_result,
                           ScalarAggregateOptions(/*skip_nulls=*/false, /*min_count=*/0));
-  ValidateMean<TypeParam>("[null]", nan_result,
-                          ScalarAggregateOptions(/*skip_nulls=*/false, /*min_count=*/0));
-  ValidateMean<TypeParam>("[]", null_result,
-                          ScalarAggregateOptions(/*skip_nulls=*/false, /*min_count=*/1));
   ValidateMean<TypeParam>("[null]", null_result,
-                          ScalarAggregateOptions(/*skip_nulls=*/false, /*min_count=*/1));
-  ValidateMean<TypeParam>(json, expected_result,
                           ScalarAggregateOptions(/*skip_nulls=*/false, /*min_count=*/0));
-  ValidateMean<TypeParam>(json, expected_result,
-                          ScalarAggregateOptions(/*skip_nulls=*/false, /*min_count=*/3));
-  ValidateMean<TypeParam>(json, expected_result,
-                          ScalarAggregateOptions(/*skip_nulls=*/false, /*min_count=*/4));
   ValidateMean<TypeParam>(json, null_result,
-                          ScalarAggregateOptions(/*skip_nulls=*/false, /*min_count=*/15));
+                          ScalarAggregateOptions(/*skip_nulls=*/false, /*min_count=*/0));
```

Comment:
It seems like there should be some tests with `skip_nulls=false` and a non-zero `min_count`?

Reply:
Added, sorry.